server/email/email-controller.js

```diff
@@ -0,0 +1,55 @@
+var mandrill = require('./email-service');
+var _ = require('underscore');
+
+module.exports = function(app, io){
+  var models = app.get('models');
+  var Gigs = models.Gig;
+  var UserGigs = models.UserGigs;
+  var Users = models.User;
+  var Locations = models.Location;
+
+  var emailController = {
+
+    sendEmails: function (req, res) {
+      var gigId = req.body.gigId;
+
+      Gigs.getGigInfo(gigId).then(function (gigInfo) {
+        Locations.getLocationInfo(gigInfo.LocationId).then(function (locationInfo) {
+          UserGigs.getUserGigs(gigId).then(function (userIds) {
+            _.each(userIds, function(userId) {
+              Users.getEmployeeEmail(userId.UserId).then(function (userInfo) {
+                mandrill.sendEmployeeConfirmationMessage(gigInfo.dataValues, userInfo.dataValues, locationInfo.dataValues);
```

Comment:
you should use the sequelize getters and setter - no need to access the dataValues directly